index.html

```diff
@@ -20,28 +20,38 @@
         <button class="hamburger">&#9776;</button>
         <div class="nav-list">
           <ul>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Projects</a></li>
-            <li><a href="#">Skills</a></li>
+            <li class="tab" data-content="aboutMe"><a href="#">About</a></li>
```

Comment:
I can't comment on the actual line itself because it's from a previous commit, but Line 16 has a spelling error and should be:

```
    <h2>Web Dveleoper Portfolio</h2>
```